.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "Earthly Docker",
+	"image": "mcr.microsoft.com/devcontainers/universal:2",
+	
+    // Installs earthly with docker and bootstraps it
+	"features": {
+        "ghcr.io/earthly/devcontainer-features/earthly:1": {
+            "bootstrap": true
+        }
+    },
+
+    // Adds on Earthly syntax highlighting extension
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"earthly.earthfile-syntax-highlighting"
+			]
+		}
+	},
+
+    // You can update your repo to pass any required secrets then load them below
+    // It is recommended to login to docker to avoid rate limiting issues
+    // For more information on setting up your secrets read the guide here:
+    // https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/configuring-dev-containers/specifying-recommended-secrets-for-a-repository
+    
+    // "secrets": {
+    //     "DOCKER_USERNAME": {
+    //       "description": "Your username for logging into docker",
+    //     },
+    //     "DOCKER_PASSWORD": {
```

Comment:
We might want to use `DOCKERHUB_USERNAME` and `DOCKERHUB_PASSWORD` for clarity. I think we use these specific names in the onboarding tutorial too.